### PR TITLE
add a link to the yaml file

### PR DESCRIPTION
**Pull request by arm61**
There was no link to the yaml file, this has been resolved.

### CODE REVIEW

**Review by nvaytet — CHANGES_REQUESTED** [review]
Can you remove the outputs in your notebook?

**Comment by arm61** [comment]
My bad!

File: reflectometry/amor_reduction.ipynb
Comment (nvaytet) [inline reflectometry/amor_reduction.ipynb:231]: If this is not really urgent, can we wait until this is merged in `ess`? https://github.com/scipp/ess/pull/23

Comment (arm61) [reply reflectometry/amor_reduction.ipynb:231]: It 100 % can wait. Just wanted to put it in when it was on the top of my head.

**Comment by nvaytet** [comment]
@arm61 now that the changes have been merged in the `ess` repo, do you want to update this?

**Comment by arm61** [comment]
I think this commit is ready now

**Comment by nvaytet** [comment]
I think my point about the `ess` repo was that the `qGrid` function could now be imported from the `ess.amor` submodule instead of writing it out in the notebook?

**Comment by arm61** [comment]
@nvaytet I think this is good to go now.

File: reflectometry/output.ort
Comment (nvaytet) [inline reflectometry/output.ort:47]: Why have the results changed? because the bins are different?
it suggests this test is going to break again in the future, and if we are always overwriting the test results, it may suggest we are not really testing much?

**Comment by nvaytet** [comment]
> @nvaytet I think this is good to go now.

Sorry i somehow missed the notification for this...

File: reflectometry/output.ort
Comment (arm61) [reply reflectometry/output.ort:47]: Looks like there are much fewer bins to me...

**Comment by arm61** [comment]
I will resolve these once #43 is merged.

File: reflectometry/amor_reduction.ipynb
Comment (nvaytet) [inline reflectometry/amor_reduction.ipynb:15]:
```suggestion
    "- `conda env create -f ess-notebooks.yml python=3.7`\n",
```
(the `-latest` file has been removed.)

File: reflectometry/amor_reduction.ipynb
Comment (nvaytet) [inline reflectometry/amor_reduction.ipynb:17]:
```suggestion
    "The [yaml environment file](https://github.com/scipp/ess-notebooks/raw/master/ess-notebook.yml) is part of this repository.\n",
```

File: reflectometry/output.ort
Comment (nvaytet) [reply reflectometry/output.ort:47]: > Looks like there are much fewer bins to me...

Are there meant to be fewer bins or is this a bug?

**Comment by arm61** [comment]
I think this is ready now...

File: reflectometry/output.ort
Comment (arm61) [reply reflectometry/output.ort:47]: I think it is a result of a change to the `q_grid` function?

Comment (nvaytet) [reply reflectometry/output.ort:47]: So you are writing this file as part of the notebook, but it is not being used after that.
So it doesn't need to be under version control, right?
Can we remove it from the codebase?

Comment (arm61) [reply reflectometry/output.ort:47]: It is linked in the notebook text so the user can download and have a peek

File: reflectometry/amor_reduction.ipynb
Comment (nvaytet) [inline reflectometry/amor_reduction.ipynb]:
```suggestion
    "The [yaml environment file](https://raw.githubusercontent.com/scipp/ess-notebooks/main/ess-notebooks.yml) is part of this repository.\n",
```

**Pull request merged**
